todo/src/components/Form.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+
+export default class Form extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            input: '',
+        }
+    }
+
+    changeHandler = e => {
+        this.setState({
+            input: e.target.value,
+        })
+    }
+
+    submitHandler = e => {
+        e.preventDefault();
+        (this.state.input !== '') &&
+        this.props.add(this.state.input);
```

Comment:
This is an interesting way to do this over just using an `if` statement; I think `if` is preferred in methods since you can use it and it's slightly more readable.